app/components/base/Card.tsx

```diff
@@ -107,14 +116,28 @@ const DescriptionStyle = styled.div`
   }
 `;
 
+export type LanguageProp =
+  | "css"
+  | "handlebars"
+  | "html"
+  | "java"
+  | "javascript"
+  | "kotlin"
+  | "php"
+  | "python"
+  | "ruby"
+  | "shell"
+  | "typescript";
+
```

Comment:
Made language options explicit to avoid spelling mistakes and typos when using the Card component.